src/server/data/mergeHelper.js

```diff
@@ -6,8 +6,24 @@ const buildLearners = () => {
     if (error) {
       throw error;
     }
-    files.forEach(file => {
+
+    let count = 0;
+    files.forEach((file) => {
       let data = require(`./learners/${file}`);
+      data.id = count;
+      count++;
```

Comment:
so there is one .... data? i'm curious what this id is for. ah, 0?

allLearners.data.skills[32] would give me a javascript object that represents a skill, right?

Reply:
The base id is for the learner. Since all the ids need to be unique I'm just starting from there also made the changes you requested I think?

Reply:
Also it's not exactly that specific, since all the ids need to be unique I made it so everything that needs an id just increments the id counter.